css/theme-topic.css

```diff
@@ -7,26 +7,26 @@
 	margin-top: 0
 }
 
-.pstn-relative {
+.pstn-relative.experimental {
   position: relative;
 }
 
-.hide-overflow {
+.hide-overflow.experimental {
   overflow-x: hidden;
 }
 
-.tt-img {
+.tt-img.experimental {
   padding-left: 0px;
   padding-right: 0px;
   margin-left: 25px;
   background-color: #f5f5f5;
 }
 
-.pagetag {
+.pagetag.experimental {
```

Comment:
I was wondering if you knew what the ccs class `pagetag` does. Because I've search in theme.css and wet-boew.css and I haven't seen this ruleset declared at all.

Reply:
Is was remove from the CSS stylesheet in WET 4.0.29

Reply:
Yeah, it's still on many Canada.ca pages and in the WET demo pages. 

I thought it would be useful to use it to set the width restriction, it's better than targetting the first p tag on the page (since this may not be mandatory for someone to have an intro paragraph)

Should we make a new class instead of using a retired class?

Reply:
Nope, we should keep it as is. It's inside a `experimental` class.

Reply:
@jmealing Let's change `.experimental` for `.provisional` in `theme-topic.css`.

Reply:
Updated to "provisional" in all the new .css files and the demo page